client/homescreen/activity-panel/orders/index.js

```diff
@@ -224,6 +224,7 @@ class OrdersPanel extends Component {
 				{ cards }
 				<ActivityOutboundLink
```

Comment:
I wonder how much value the `ActivityOutboundLink` component really has... this is the only usage of it. Why don't we just get rid of it?

Reply:
Looking at the fact that you're adding more code to this unused component to remove the icon rendering I agree that ditching it has some value here. It has a bit of styling associated with it, but that styling could just move to a class name on a `<Link>` that you render here instead? It would be great to keep removing unused code to trim the code base down.

Reply:
Good idea! I changed it in the commit 57ae4f5